test/WebSites/ValidationWebSite/bower.json

```diff
@@ -1,21 +0,0 @@
-{
-    "name": "ValidationWebSite",
-    "description": "Web site demonstrating various validations.",
-    "private": true,
-    "dependencies": {
-        "jquery": "1.11.2",
-        "jquery-validation": "1.13.1",
-        "jquery-validation-unobtrusive": "3.2.2"
```

Comment:
To confirm: Decision was to leave the `[RemoteAttribute]`s now in `BasicWebSite` unusable even when debugging the site or otherwise using a browser?